.travis.yml

```diff
@@ -8,6 +8,14 @@ matrix:
       env: SYMFONY_VERSION='2.3.*'
     - php: 5.6
       env: SYMFONY_VERSION='2.5.*'
+    - php: 5.6
+      env: SYMFONY_VERSION='2.6.*'
+    - php: 5.6
+      env: SYMFONY_VERSION='2.7.*'
+    - php: 5.6
+      env: SYMFONY_VERSION='2.8.*'
+    - php: 5.6
+      env: SYMFONY_VERSION='3.0.*'
```

Comment:
you should limit the number of builds here. Symfony 2.5 and 2.6 are unmaintained